t/test.pl

```diff
@@ -1435,7 +1438,7 @@ ($@)
 sub new_ok {
     my($class, $args, $obj_name) = @_;
     $args ||= [];
-    $object_name = "The object" unless defined $obj_name;
+    my $object_name = "The object" unless defined $obj_name;
```

Comment:
Conditional my declaration, will lead to bugs.

Reply:
actually this looks like a bug on closer inspection. `s/object_name/obj_name/g`

Reply:
I agree, it doesn't really make sense as written, this and the below usage likely should be using `$obj_name`.

Reply:
Fixed.